src/MergeContext.js

```diff
@@ -234,7 +229,7 @@ class StatusChecks
 
     // no more required status changes or additions are expected
     final() {
-        return (this.requiredStatuses.length >= this.expectedStatusCount) &&
+        return (this.requiredStatuses.length >= this._contextsRequiredByGitHubConfig.length) &&
```

Comment:
this.requiredStatuses.length should be <= this._contextsRequiredByGitHubConfig.length because this.requiredStatuses only gets statuses that are among this._contextsRequiredByGitHubConfig. So we can probably assert the '<=' condition here and use '==' instead for the returning condition.